api/src/queries/project/project-view-queries.test.ts

```diff
@@ -13,6 +11,8 @@ import {
   getRegionsByProjectSQL
 } from './project-view-queries';
 
+import { getFocalSpeciesByProjectSQL, getAncillarySpeciesByProjectSQL } from './project-view-update-queries';
```

Comment:
These are now being tested in the project-view-update-queries.test.ts, so we can remove duplicate tests from this file.